src/button.hpp

```diff
@@ -0,0 +1,28 @@
+#include <SFML/Graphics.hpp>
+
+class Button {
+public:
+	Button (std::string textString,
+		   	sf::RenderWindow & win,
```

Comment:
TODO: it's not so convenient to pass the same value to every button object. It's not urgent, so I think we'll keep it at the moment, but we need to change it some day.